features/testbot/distance_matrix.feature

```diff
@@ -100,3 +100,30 @@ Feature: Basic Distance Matrix
             | y | 500 | 0   | 300 | 200 |
             | d | 200 | 300 | 0   | 300 |
             | e | 300 | 400 | 100 | 0   |
+
+    # There is a 1100m limit when searching for nearest neighbours
```

Comment:
I plan to remove this limit completely. It seldom hits anyway and it has been a constant source of grievance while debugging.